src/header/styles.scss

```diff
@@ -103,6 +103,10 @@
 }
 
 .title {
+  display: flex;
```

Comment:
Remembering the story with flexible containers (https://github.com/cloudscape-design/components/pull/603) can we make extra flex-styles opt-in for the new mode to reduce possible impact?

Reply:
Yes, I think it is better to do that.

Ideally is to have fewer conditions in the code but I agree it is better to decrease the blast radius. I am still waiting for the screenshot test results to see if there are noticeable diffs.